package.json

```diff
@@ -10,12 +10,16 @@
   "homepage": ".",
   "license": "MIT",
   "dependencies": {
+    "@date-io/date-fns": "2.10.8",
+    "@date-io/moment": "1.x",
```

Comment:
Pin exact version.

Reply:
In the installation guide, it is required to use this version for date-pickers https://material-ui-pickers.dev/getting-started/installation
```For material-ui-pickers v3 use v1.x version of @date-io adapters.```